test/hyperdeck/response-handler.js

```diff
@@ -14,38 +14,23 @@ var SUCCESS_RESPONSE_EVENT_PAYLOAD = {
   }
 };
 
-// See format response
-var SUCCESS_RESPONSE_WITH_DATA_NO_BUT_NOT_PARAMS = "200 Success with data but not params and no colon\r\nabc\r\n\r\n";
```

Comment:
I think it would be good to keep this as it is how the format command works with the token, but change the status code to 201 or something

Reply:
Actually I think some of the tests have gone missing that test lines arriving separately. I'm happy to figure out where these went and put them back when we merge this :)

Reply:
👍